packages/grafana-data/src/types/config.ts

```diff
@@ -236,6 +236,9 @@ export interface GrafanaConfig {
   reportingStaticContext?: Record<string, string>;
   exploreDefaultTimeOffset?: string;
 
+  // Basic actions configuration
+  actions?: { allowPostURL?: string };
```

Comment:
should we revert this and the grafana-runtime change?

Reply:
yep, same here: https://github.com/grafana/grafana/pull/91822/files#diff-8e6d1dc8bffebb9fc6d982a51a284e963b36aefdd18a58451fff8abb47068db5R194

Reply:
cleaned this up in recent commit